### PR TITLE
Fix Page Not Found error

### PR DESCRIPTION
**Pull request by lnnt**

### CODE REVIEW

File: docs/content/installation/from-source.zh-cn.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ git checkout v@version@
 
 - `go` @minGoVersion@ 或以上版本, 详见[这里](https://golang.google.cn/doc/install)
 - `node` @minNodeVersion@ 或以上版本，并且安装 `npm`, 详见[这里](https://nodejs.org/zh-cn/download/)
-- `make`, 详见[这里](/zh-cn/hacking-on-gitea/)
+- `make`, 详见[这里](/zh-cn/development/hacking-on-gitea/)
```
Comment (lunny) [inline docs/content/installation/from-source.zh-cn.md]:
```suggestion
- `make`, 详见[这里](development/hacking-on-gitea.md)
```

Comment (lnnt) [reply docs/content/installation/from-source.zh-cn.md]: https://docs.gitea.com/development/hacking-on-gitea.md is incorrect URL

**Review by lunny — APPROVED** [review]

File: docs/content/installation/from-source.zh-cn.md (hunk 1 above)
Comment (lunny) [reply docs/content/installation/from-source.zh-cn.md]: The link will be converted

Comment (xpume) [reply docs/content/installation/from-source.zh-cn.md]: https://docs.gitea.com/development/hacking-on-gitea.md is incorrect URL
https://docs.gitea.com/development/hacking-on-gitea is correct URL
Right?

**Comment by yp05327** [comment]
image
There's another one

image
image
It seems that both of them are incorrect.

**Review by delvh — APPROVED** [review]

**Pull request merged**